src/init.cpp

```diff
@@ -1532,6 +1532,9 @@ bool AppInitMain(NodeContext& node)
             try {
                 LOCK(cs_main);
                 g_chainman.InitializeChainstate();
+                g_chainman.m_total_coinstip_cache = nCoinCacheUsage;
+                g_chainman.m_total_coinsdb_cache = nCoinDBCache;
```

Comment:
```suggestion
                chainman.m_total_coinsdb_cache = nCoinDBCache;
```

Needs rebase to remove the `g_` prefix in this line

Reply:
Pushed, thanks!